garrysmod/gamemodes/terrortown/gamemode/lang/english.lua

```diff
@@ -217,7 +217,7 @@ L.sb_deaths     = "Deaths"
 L.sb_score      = "Score"
 L.sb_karma      = "Karma"
 
-L.sb_info_help  = "Search this player's body, and you can review the results here."
+L.sb_info_help  = "Search this player's body to review the results here."

```

Comment:
Pointless change

Reply:
Just a suggestion to make it a bit convenient for other localizations. I will change my localization if this will be declined.